src/main/java/com/iprd/fhir/utils/FhirResourceTemplateHelper.java

```diff
@@ -132,6 +157,20 @@ public static Location clinic(String state, String district, String city, String
 		facility.setStatus(LocationStatus.ACTIVE);
 		facility.setMode(LocationMode.INSTANCE);
 		facility.setPhysicalType(facilityPhysicalType);
+		if (!latitude.equals("null") && !longitude.equals("null")){
+			Location.LocationPositionComponent position = new Location.LocationPositionComponent();
+			position.setLongitude(Double.parseDouble(longitude));
+			position.setLatitude(Double.parseDouble(latitude));
```

Comment:
I think this parser is likely to throw an error if the user provide non numeric values. Should be handled by try catch block.

Reply:
Yes we should add a check just incase there is wrong input and the upload for this clinic should fail with the right error message sent to the user